sda/cmd/auth/oidc.go

```diff
@@ -17,6 +17,7 @@ import (
 // OIDCIdentity represents an OIDC user instance
 type OIDCIdentity struct {
 	User                 string
+	Name                 string
```

Comment:
What is `Name` for?

Reply:
It is the name of the user, eg `John Franklin Doe`. To be saved in the db:
```
id            |       name        |        email         |                              groups                               
-------------------------+-------------------+----------------------+-------------------------------------------------------------------
jd123@lifescience-ri.eu | John Franklin Doe | j.f.doe@faberuni.edu | {urn:geant:lifescience-ri.eu:group:researchers#lifescience-ri.eu}
```